tests/wasmfs/wasmfs_opfs.c

```diff
@@ -0,0 +1,119 @@
+#include <assert.h>
```

Comment:
Do we have an existing backend test template? It seems like we could use this file, or one like it if we already have one, and in each backend just include the file, maybe using some ifdefs to control things (like disable symlinks in OPFS since it lacks them). Otherwise the core testing code for each backend seems like it would be very similar.

Reply:
I don't think we have a template yet and I agree this file would make a good template. I don't think anything should change for this PR, though. We can add those ifdefs once we're ready to test the node backend like this perhaps.